media-video/mpv/mpv-0.38.0.recipe

```diff
@@ -98,9 +99,10 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	ffmpeg5${secondaryArchSuffix}_devel
+	ffmpeg6${secondaryArchSuffix}_devel
```

Comment:
This should use `devel:` entries instead (see in `REQUIRES` which ones). ffmpeg5 didn't provide them, which is why this used the package name instead, but ffmpeg6 does.